src/components/Slider/RangeSlider/RangeSlider.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+import {
+  Root as SliderRoot,
+  Track as SliderTrack,
+  Range as SliderRange,
+  Thumb as SliderThumb
+} from '@radix-ui/react-slider';
+
+import styles from '../Slider.module.scss';
+import classNames from 'classnames/bind';
+const cx = classNames.bind(styles);
+
+export interface RangeSliderProps {
+  values?: number[];
+  min: number;
+  max: number;
+  step: number;
+  minStep?: number;
+  isSmall?: boolean;
+  isLight?: boolean;
+}
+
+export const RangeSlider: React.FC<RangeSliderProps> = ({ step, min, max, values, isSmall, minStep, isLight }) => {
+  const [currentValues, setCurrentValues] = useState(values ?? [min, max]);
+  return (
+    <div className={cx(styles.RootContainer, { LightRootContainer: isLight })}>
+      {currentValues[0]} - {currentValues[1]}
+      <form>
+        <SliderRoot
+          className={cx(styles.SliderRoot, { SmallRoot: isSmall, LightRoot: isLight })}
+          defaultValue={[min, max]}
+          min={min}
+          max={max}
+          step={step}
+          minStepsBetweenThumbs={minStep}
+          aria-label="Range slider"
+          onValueChange={value => setCurrentValues(value)}
+        >
+          <SliderTrack className={styles.SliderTrack}>
+            <SliderRange className={cx(styles.SliderRange, { SmallRange: isSmall, LightRange: isLight })} />
+          </SliderTrack>
+          <SliderThumb className={cx(styles.SliderThumb, { SmallThumb: isSmall, LightThumb: isLight })} />
+          <SliderThumb className={cx(styles.SliderThumb, { SmallThumb: isSmall, LightThumb: isLight })} />
```

Comment:
`cx(styles.SliderThumb, { SmallThumb: isSmall, LightThumb: isLight })` is repeated - best to set it as a variable in the function body imo.

```ts
const thumbClasses = cx(styles.SliderThumb, { SmallThumb: isSmall, LightThumb: isLight });

return  <SliderThumb className={thumbClasses} />
```

However, I think it's best to handle these cases as custom attributes rather than classes, e.g.
```ts
// component
<SliderThumb data-size={isSmall ? "small" : "default"} />

// selecting in scss
.SliderThumb[data-size="small"] { ... }
```
or similar.